packages/components/src/internal/components/security/APIWrapper.ts

```diff
@@ -215,6 +216,22 @@ export class ServerSecurityAPIWrapper implements SecurityAPIWrapper {
 
     getUserLimitSettings = getUserLimitSettings;
 
+    getUserPermissions = (options: Security.GetUserPermissionsOptions): Promise<string[]> => {
+        return new Promise((resolve, reject) => {
+            Security.getUserPermissions({
```

Comment:
Just to note there is already a specific wrapper for the "/Shared" folder that calls this same endpoint wrapper. It's called `getUserSharedContainerPermissions`. We should consider consolidating their usages at some point. Not worth doing as a part of this PR.